api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/OpenEmpiPatientFetcher.java

```diff
@@ -36,15 +38,20 @@
     private RegistrationCoreProperties coreProperties;
 
     @Override
-    public Patient fetchMpiPatient(String patientId) {
-        OpenEmpiPatientResult mpiPatient = queryExecutor.getPatientById(authenticator.getToken(), patientId);
+    public MpiPatient fetchMpiPatient(PatientIdentifier patientId) {
+        OpenEmpiPatientResult mpiPatient = queryExecutor.getPatientById(authenticator.getToken(), patientId.getIdentifier());
```

Comment:
Actually this is the type I expect the fetchMpiPatient method to return from an earlier method instead of also having MpiPatient

Reply:
Yes the earlier class has been changed

Reply:
So why do you still have both, I'd expect just one